pkg/eval/math/math.go

```diff
@@ -121,7 +121,7 @@ import (
 //elvdoc:fn is-nan
 //
 // ```elvish
-// math:is-nan $number $sign
+// math:is-nan $number
```

Comment:
Document the `&sign` option.

Reply:
The point of this change is to remove the explicit `$sign` argument, as well as the implicit `&sign` option, from the documentation since neither is relevant, or valid, for arguments passed to `math:is-nan`. The original line was a copy/paste mistake from the `math:is-inf` implementation.

Reply:
Are you suggesting that `math:is-nan` explicitly support a `&sign` option? Even though that option is meaningless with respect to NaN values?

Reply:
Ah, I mistook this for `is-inf`.